### ISSUE TITLE
.vimrc isn't being parsed with 'let' expression

### ISSUE THREAD

**Issue by eddieparker** [VsVim/VsVim#2860]
**Describe the bug**
My vimrc is getting ignored.

**To Reproduce**
The following .vimrc:

```
let HOME_DIR="$EDROPBOX_HOME\\Utilities\\MSYS2Portable\\App\\msys32\\home\\eddie"
exe "set runtimepath+=" . HOME_DIR . "\\.vim"
exe "source " . HOME_DIR . "\\.vimrc"
```

Yields:

![image](https://user-images.githubusercontent.com/1141487/98718533-cd5ee600-234b-11eb-8921-a6ab1cf344f1.png)

**Expected behavior**
Ideally it would work for sourcing the vimrc in question.

**Additional context**

I realize also that vsvim likely won't support *everything* that vim does - is there a way to detect in a .vimrc file that I'm loading .vsvim, so I can run different code there?

**Comment by WrGraban** [comment]
I agree with @eddieparker, searching for a way to detect if the vimrc file can detect if it is being loaded by vsvim

**Comment by mqudsi** [comment]
I think you can create a `.vsvimrc` file that defines a global variable `let g:is_vsvim=1` then sources your shared `.vimrc` file (`:source .vsvimrc`) which lets you conditionally enable/disable logic in your shared vsvim config (`if exists("g:is_vsvim") ... `)
